Show and allow creation of unknown performers/tags/studios/movies from scraper dialog

WithoutPants opened the pull request:
Shows performers/tags/studios/movies that are unknown to the stash database under the applicable entry in the scraper dialog, with a button to create and add the object to the new value.

![image](https://user-images.githubusercontent.com/53250216/90329934-3bfe0300-dfec-11ea-8ff7-c5ae6b87e37d.png)

I'm not sold on the presentation, so let me know if there's a better way to present this information.

Clicking on the + button removes the entry from the list and adds it to the applicable field.

Also changes the scraping operations so that they do not cache the results.

bnkai commented:
Testing seems ok.
Not sure on how it would look better but as a first note the tags section should probably be last in order as some sites use a lot of tags (30+). A suggestion from the discord channel for the tags section was for something like a checkbox to show/hide tags that are not present in the DB (switch between dev behaviour and the new from this PR).

WithoutPants commented:
I've changed the missing tags to use clickable badges. I've also added a collapse element that hides the missing tags if there are >= 10 tags.

![image](https://user-images.githubusercontent.com/53250216/90473924-6fb16800-e167-11ea-9d59-b6fd95aea7bb.png)

![image](https://user-images.githubusercontent.com/53250216/90473937-79d36680-e167-11ea-98e9-0719696f226b.png)
